launchcontainer/launchcontainer.py

```diff
@@ -298,7 +298,7 @@ def studio_submit(self, data, suffix=''):
             self.project = data['project'].strip()
             self.project_friendly = data['project_friendly'].strip()
             self.project_token = data['project_token'].strip()
-            self.support_email = data['support_email'].strip()
+            self.support_url = data['support_url'].strip()
             self.api_url = self.wharf_url
```

Comment:
Step 3) We can hardcode support URL to `/help` because it's a standard Open edX URL.